### ISSUE TITLE
Profiles for my profiles 

### ISSUE THREAD

**Issue by never147** [shupershuff/Diablo2RLoader#12]
So I really like the different settings option! What might be really cool is if I could choose which settings profile to use when opening an account. So, I have 3 accounts, 3 separate settings files, but I only have 2 different types of settings. 1 for full screen detailed graphics and 1 for windowed low graphics. Sometimes I switch which account is the one using the high graphics and the others use the low. It would be cool if I could choose that in the loader. So perhaps another field in the account file with the default settings profile number and a question when loading, similar to the realm one, for selecting the profile at load. I guess you could have an option in the defaults to turn this question off if you don't use the feature.

Thanks!

**Comment by shupershuff** [comment]
A great suggestion and something I've come across myself as sometimes you want your primary to look nice-ish and have a decent framerate while the alt accounts need to run as lightweight as possible.
Good odds you'll see this in a future release, not sure how I will go about it but it would possibly require changes in config.xml, accounts.csv as well as user input from the menu.

A great idea so thanks for the feature request.

**Comment by never147** [comment]
Thank you for being receptive to the idea!

**Comment by shupershuff** [comment]
So I want to keep the amount of inputs required to launch the game as short as possible.
As such my rough thoughts are that I add an option in the first menu "s" to toggle manual settings changer. Disabled by default, it will load game settings from settings1.json for account 1, settings2 for account 2 etc (as it already does today). 

If enabled, it would ask you to specify a settings file for each account, with the list of options being populated by any settings*.json file in the saved games folder that isn't settings.json or settings<_id_>.json.
In other words, you'd make config files such as settingshigh.json or settings.woodengraphicscard.json and in the menu if would display any text between "settings" and ".json" as the options to choose from. If no such files are found, an error message will be shown with instructions to set it up.

Note that this feature wouldn't work great if you choose to launch all accounts, as it would need to ask for input for each account.
There would be no changes in config.xml or accounts.csv required.

**Comment by never147** [comment]
That sounds cool. Yeah no worries about the timing. Thanks for looking into it! Good luck with the grind to 99!!

**Comment by shupershuff** [comment]
Should have this published tomorrow, pending some testing :)

**Comment by shupershuff** [comment]
Aaaaand Published. Enjoy! Let me know if there's any issues :)

**Comment by never147** [comment]
This is very cool. Thanks for spending the time on it! 😍
